### ISSUE TITLE
Solve with inequalities

### ISSUE THREAD

**Issue by Happypig375** [jiggzson/nerdamer#251]
Currently the solve functions (solve, solveFor, solveEquations) only accept equations. I propose redefining the existing comparison operators to creating inequalities and letting the solve functions solce them.

**Comment by jarble** [comment]
In some cases, inequalities can be re-written as equations using the [sign function](https://en.wikipedia.org/wiki/Sign_function). `x>0` could be written as `sqrt(x)/x = 1`.

**Comment by jiggzson** [comment]
@jarble, thanks for the input. Definitely something to consider. Does `x<0` then become `-sqrt(x)/x = 1`? This might be a silly question since I just skimmed the article but would be a rewrite for `x>=0`? Do you break it up into two equations at this point?

**Comment by jarble** [comment]
Yes, `x<0` could be also be written as `sqrt(x)/x = -1`. `sqrt(x)/x` is undefined when x = 0, so `x >= 0` would need to be written in another way.

**Comment by jarble** [comment]
`sqrt(x^2)` can also be written as `abs(x)`.

**Comment by jarble** [comment]
It seems that `x != 0` could be written as `abs(x)/abs(x) = 1`, since this expression is undefined when `x=0`.
